openlibrary/solr/update_work.py

```diff
@@ -1491,8 +1497,11 @@ def update_keys(
     :param bool commit_way_later: set to true if you want to add things quickly and add
         them much later
     """
+    start = timer()
     logger.debug("BEGIN update_keys")
     commit_way_later_dur = 1000 * 60 * 60 * 24 * 5  # 5 days?
+    timer_results = f"RESULTS OF TIMER FOR KEYS LIKE {keys[0]}" \
+                    f" with {number_threads} threads \n"
```

Comment:
PEP8: Please avoid backslash line continuation in Python.
```suggestion
    timer_results = (f"RESULTS OF TIMER FOR KEYS LIKE {keys[0]}"
                    f" with {number_threads} threads \n")
```